prisma/schema.prisma

```diff
@@ -91,6 +91,18 @@ model FIR {
   groups   Group[]  @relation("FIRtoGroups")
 }
 
+// Client Deliverables
+model Deliverables {
+  deliverableName String  @unique // "groupData" or "datasetTimestamps"
+  content         String          // stringified json to send to the client
+}
+
+// Deliverable Timestamps
+model Timestamps {
```

Comment:
Should be hashes/keys not timestamps, to represent versioning. Client should store the hash locally in state, and use *that* for version comparison. Timestamps will constantly change and for our purposes a single version hash/id is sufficient.

Reply:
Call it whatever you want 🤷‍♂️ 

I do think the date context is still potentially helpful, and it works as a guaranteed-unique value. For clarity, are you saying to rename it, or to not use a Date.now() value?

Reply:
Ah, I think I misunderstood the way in which the version-comparison would work, which perhaps suggests we could find a more descriptive/clear name (or document it more thoroughly).

I initially thought that two different times were being compared to check version compatibility (time of check with the time of update, which didn't make much sense). What we intend to do is pass that timestamp to the client and then have the client use that as the "last retrieved info updated at _X_ from server, does server have anything > _X_?"

Is that understanding correct?

Reply:
Essentially, though in reverse. My thoughts was:
- client makes a request for new data, and includes as a part of that request a simple object like `{ facilityData: 12345, group data: 34567 }`, to which the API would respond with:
    - "up to date", or
    - new group data set, or
    - new facility data set, or
    - new group + facility data sets

Reply:
...noting that the data sets include the timestamp within them. So by the API sending new group data, the client will then make their next request using that timestamp.